darwin/future/meta/queries/stage.py

```diff
@@ -14,7 +14,7 @@ def _collect(self) -> List[Stage]:
             raise ValueError("Must specify workflow_id to query stages")
         workflow_id: UUID = self.meta_params["workflow_id"]
         meta_params = self.meta_params
-        workflow, exceptions = get_workflow(self.client, str(workflow_id))
+        workflow = get_workflow(self.client, str(workflow_id))
```

Comment:
Maybe need handling of the exceptions here?  Consideration point

Reply:
good point, I'll add something in. At least a print on failed validations